test/core/surface/public_headers_must_be_c89.c

```diff
@@ -74,5 +74,18 @@
 #include <grpc/support/time.h>
 #include <grpc/support/tls.h>
 #include <grpc/support/useful.h>
+#include <grpc_c/bidi_streaming_blocking_call.h>
```

Comment:
Exclude these headers in the template code: no need for the C wrapper to insist upon C89.